common/djangoapps/django_sudo_helpers/templates/sudo/third_party_auth_links.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n third_party_auth_links %}
+<script type="text/javascript">
+    function thirdPartySignin(event, url) {
+      event.preventDefault();
+      window.location.href = url;
+    }
+</script>
+<span class="deco-divider" style="background: transparent;">
+    <span class="copy">{% trans "or" %}</span>
+</span>
+
+<div class="form-actions form-third-party-auth">
+
+{% for enabled in providers %}
+    <button type="submit" class="button button-primary button-{{enabled.provider_id}} login-{{enabled.provider_id}}" {% if enabled.has_account %} onclick="thirdPartySignin(event, '{{pipeline_url|get_pipeline_url:enabled.provider_id}}');" {% else %} disabled="disabled" style="color: #ffffff;" {% endif %} ><span class="icon fa {{enabled.icon_class}}"></span>{% trans 'Sign in with' %} {% trans enabled.name %}</button>
```

Comment:
The `trans` calls on this line will also cause problems for our translators.

1) They will receive the message "Sign in with" which doesn't give them any context. Sign in with what? I suggest using a format string so that they can see something like "Sign in with {provider_name}". See [this part](https://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#don-t-append-strings-interpolate-values) of the i18n guide.

2) The `trans` call around the `enabled.name` value will not work unless `enabled.name` has been passed into a  `uggettext_noop` call earlier.